apps/src/applab/applab.js

```diff
@@ -1491,6 +1513,7 @@ Applab.execute = function() {
       'if (obj) { var ret = obj.fn.apply(null, obj.arguments ? obj.arguments : null);' +
                  'setCallbackRetVal(ret); }}';
     var session = studioApp.editor.aceEditor.getSession();
+    annotationList.attachToSession(session);
```

Comment:
I mostly did it twice so that I didn't have to worry about race conditions. I think the right approach is the strict equality check so that subsequent calls are no-ops, but we still don't have to worry about potential race conditions.